### ISSUE TITLE
Login & registration form offset

### ISSUE THREAD

**Issue by xiaoyong-wang** [go-gitea/gitea#16815]
- Gitea version (or commit ref): 1.15.0
- Git version: 2.25.1
- Operating system: Ubuntu 20.04.3 LTS
I am running gitea using the prebuilt binary from dl.gitea.io. It is running via systemd.
  
  
  
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [X] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [X] No
- Log gist:
- N/A

## Description

The login & registration form appears to be offset (to the right) on mobile. You can see in the screenshots below.

## Screenshots
(This is on mobile. On desktop, this issue does not exist.)
https://i.imgur.com/gnFQ9Ob.png
https://i.imgur.com/Jv3bbci.png

**Comment by silverwind** [comment]
Can't reproduce on desktop browser in mobile mode at least:

<img width="384" alt="image" src="https://user-images.githubusercontent.com/115237/132262758-2b0de0d6-ada6-4f57-9d2e-7329c0dddce1.png">
<img width="390" alt="image" src="https://user-images.githubusercontent.com/115237/132262783-3e6cd6e7-21a1-4fc3-b4b2-ec65cc470ce9.png">

**Comment by kasperireland** [comment]
> Can't reproduce on desktop browser in mobile mode at least:
> 
> <img alt="image" width="384" src="https://user-images.githubusercontent.com/115237/132262758-2b0de0d6-ada6-4f57-9d2e-7329c0dddce1.png">
> 
> <img alt="image" width="390" src="https://user-images.githubusercontent.com/115237/132262783-3e6cd6e7-21a1-4fc3-b4b2-ec65cc470ce9.png">

As mentioned, this issue is not present on Gitea.com
It is present with pretty much any other Gitea instance, including codeberg.org

**Comment by silverwind** [comment]
I see it on https://codeberg.org/user/sing_up. Issue comes from this rule:

```css
@media only screen and (max-width: 767.98px) {
  .ui.ui.ui.container:not(.fluid) {
    width: auto;
    margin-left: 1em;
    margin-right: 1em;
  }
}
```

No idea why but that rule does not seem to be present on try.

**Comment by silverwind** [comment]
Actually this was fixed in https://github.com/go-gitea/gitea/pull/16396, will backport it.

**Comment by kasperireland** [comment]
> I see it on https://codeberg.org/user/sing_up. Issue comes from this rule:
> 
> ```css
> @media only screen and (max-width: 767.98px) {
>   .ui.ui.ui.container:not(.fluid) {
>     width: auto;
>     margin-left: 1em;
>     margin-right: 1em;
>   }
> }
> ```
> 
> No idea why but that rule does not seem to be present on try.

The issue also occurs on my personal gitea instance and some others. Not sure why.

**Comment by kasperireland** [comment]
> Actually this was fixed in #16396, will backport it.

Alright, thanks.

**Comment by silverwind** [comment]
Backport to 1.15 in https://github.com/go-gitea/gitea/pull/16976.
